feat: show qr-code with api key

pedrolamas opened the pull request:
Fixes #411 

Adds new dependency to [qrcode.vue](https://www.npmjs.com/package/qrcode.vue)

![image](https://user-images.githubusercontent.com/85504/154588029-2f4fc080-8627-43b6-a5c4-8c20b7effb17.png)

Signed-off-by: Pedro Lamas <pedrolamas@gmail.com>

matmen commented on src/components/ui/AppQrCode.vue:
```diff
@@ -0,0 +1,25 @@
+  <qrcode-vue
+    :value="value"
+    :size="size"
+    level="H"
```
An error correction of `L` probably suffices here, considering this will be displayed on a monitor and not printed out where it could be damaged. IMO it makes the QR code look a bit nicer, but I'm not sure it matters

matmen commented on src/components/ui/AppQrCode.vue:
```diff
@@ -0,0 +1,25 @@
+<template>
+  <qrcode-vue
```
A (white) border around the QR code would be nice, my phone failed to pick it up because the position marks had too low of a contrast compared to the background. The qrcode.vue docs mention a [`margin` property](https://github.com/scopewu/qrcode.vue#margin), but it seemed to not work in my testing. Giving it an 8px CSS padding with a background-color of white seemed to work fine.

pedrolamas commented:
@matmen thanks for reviewing, I've updated the PR following up your on comments (also updated the main image on the PR description to show the latest changes)

Clon1998 commented:
Greate work!
Thanks, this will make setting up a printer in my app way easier :)
